Taskfile.yaml

```diff
@@ -144,6 +153,12 @@ tasks:
     status:
       - helm search repo -r '\vmetallb/metallb\v' | grep metallb
 
+  tool:install-kube-linter:
```

Comment:
Let's install this with nix as it's how we're handing all other deps right now. There's a big ol' block of tools.  you can add [`pkgs.kube-linter`](https://search.nixos.org/packages?channel=23.11&show=kube-linter&from=0&size=50&sort=relevance&type=packages&query=kube-linter) and it'll just work ™️

Reply:
is it ok for me to leave this task file, or just remove it?

Reply:
In the interest of consistency, let's remove it. If `nix develop` isn't a pleasant experience there are myriad ways to otherwise provide access to them. nix ensures that our tools are pinned to the same version and work on most conceivable environments.